starter_code/queries.md

```diff
@@ -4,76 +4,150 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+query: {name: 'Babelgum'}
+projection = {name: 1, _id: 0}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+query: {number_of_employees: {$gt: 5000}}
+project: 
+sort: {number_of_employees: 1}
+skip:
+limit: 20
+
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+query: {founded_year: {$lte: 2005, $gte: 2000}}
+project: {name: 1, founded_year: 1, _id: 0}
+sort:
+skip:
+limit:
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+query: {'ipo.valuation_amount': {$gte: 100000000}, founded_year: {$lte: 2000}}
```

Comment:
In this case, you should use the $and
```ja
 {$and: [{'ipo.valuation_amount': {$gte: 10000000}}, {founded_year:{$lt: 2010}}]}
```

Reply:
Could you explain why that is better? If I test this query in Compass, it works perfectly and gives me the correct results 🤷‍♂️
Thanks! 😊